src/platform/plugins/shared/data_view_management/public/components/delete_data_view_flyout/delete_data_view_flyout_content.tsx

```diff
@@ -0,0 +1,235 @@
+                  values: { maxRelationships: MAX_DISPLAYED_RELATIONSHIPS },
+                })}
+              </EuiText>
+              <EuiSpacer size="xs" />
+            </>
+          )}
+          <EuiBasicTable items={relationships[id]} columns={relationsColumns} />
+        </div>
+      );
+      itemIdToExpandedRowMapValues[id] = relationsTable;
+      setReviewedItems(new Set(reviewedItems).add(id));
+    }
+
+    setItemIdToExpandedRowMap(itemIdToExpandedRowMapValues);
+  };
+
+  const columns: Array<EuiTableFieldDataColumnType<RemoveDataViewProps>> = [
+    {
+      field: 'name',
+      name: dataViewColumnName,
+      sortable: true,
+    },
+    ...(hasSpaces
+      ? [
+          {
+            field: 'namespaces',
+            name: spacesColumnName,
+            sortable: true,
+            width: '100px',
+            align: 'right',
+            render: (namespaces: string[]) =>
+              namespaces.indexOf('*') !== -1 ? all : namespaces.length,
+          } as EuiTableFieldDataColumnType<RemoveDataViewProps>,
+        ]
+      : []),
+    {
+      field: 'id',
+      align: 'right',
+      isExpander: true,
+      name: (
+        <EuiScreenReaderOnly>
+          <span>
+            {i18n.translate('indexPatternManagement.dataViewTable.viewRelationships', {
+              defaultMessage: 'View relationships',
+            })}
+          </span>
+        </EuiScreenReaderOnly>
+      ),
+      render: (id: RemoveDataViewProps['id']) => {
+        const itemIdToExpandedRowMapValues = { ...itemIdToExpandedRowMap };
+
+        return (
+          <>
+            {relationships[id]?.length ? (
+              <EuiButtonEmpty
+                css={{ width: 'auto' }}
+                onClick={() => toggleDetails(id)}
+                aria-label={itemIdToExpandedRowMapValues[id] ? 'Collapse' : 'Expand'}
+                color="danger"
+              >
+                <EuiFlexGroup alignItems="center" justifyContent="flexEnd" gutterSize="s">
+                  <EuiText>
+                    {i18n.translate('indexPatternManagement.dataViewTable.review', {
+                      defaultMessage: 'Review',
+                    })}
+                  </EuiText>
+                  <EuiSpacer size="xs" />
+                  <EuiIcon type={itemIdToExpandedRowMapValues[id] ? 'arrowDown' : 'arrowRight'} />
+                </EuiFlexGroup>
+              </EuiButtonEmpty>
+            ) : (
+              <></>
+            )}
+          </>
+        );
+      },
+    },
+  ];
+
+  views = views.map((view, index) => ({
+    ...view,
+    name: view.getName(),
+    relationships: relationships[index],
+  }));
+
+  const spacesWarningText = i18n.translate('indexPatternManagement.dataViewTable.deleteWarning', {
+    defaultMessage:
+      'Deleting a data view affects every saved object that uses it, and it is deleted from every space it is shared in. This action cannot be undone.',
```

Comment:
```suggestion
      'Deleting a data view permanently removes it from all spaces. This action cannot be undone.',
```
Is it necessary to call out the saved objects if no relationship is detected?

Alternatively if we want to expand on the consequences rather than talk technically about objects: 
`Deleting a data view permanently removes it from all spaces and prevents all users from using it in any Kibana app such as Discover, Dashboards, or Lens. This action cannot be undone.`